evap/grades/templates/grades_base.html

```diff
@@ -7,5 +7,9 @@
 
 {% block subtitle %}
     {{ block.super }}
-    <li><a href="{% url "grades:index" %}">{% trans "Grade publishing" %}</a></li>
+    {% if is_grade_publisher %}
+    	<li><a href="{% url "grades:index" %}">{% trans "Grade publishing" %}</a></li>
+    {% else %}
+		<li>{% trans "Grade publishing" %}</li>
```

Comment:
we could also allow staff users to see the grades index and semester_view pages. they don't really need them and if they can access them they should also get a menu entry for that. that would mean even more menu items...
what do you think?